setup.py

```diff
@@ -26,7 +27,7 @@
     install_requires=[
         "six",
         "numpy",
-        "scipy",       # for phash
+        'scipy',       # for phash
```

Comment:
maybe revert this to double-quotes

Reply:
to be consistent with the others and also it will then not show up in the diff

Reply:
Done! And all good on my side. Feel free to squash when merging